externs/shaka/player.js

```diff
@@ -865,6 +866,10 @@ shaka.extern.DashManifestConfiguration;
  *   segments, but the seek window will be zero-sized, to be consistent with
  *   Safari. If this is false, the seek window will be the entire duration.
  *   <i>Defaults to <code>true</code>.</i>
+ * @property {number} liveSegmentsDelay
+ *   The live edge will be calculated from segments. This number is the number
```

Comment:
How about "The default presentation delay will be calculated as a number of segments.  This is the number of segments for this calculation." ?

Reply:
Done!